src/components/Messages/Messages.js

```diff
@@ -106,7 +102,7 @@ class Messages extends Component {
                 messages={messages.map(message => ({
                   ...message,
                   user: users
-                    ? users[message.userId]
+                    ? users[message.userId] || {}
```

Comment:
I think the change is not necessary, no? Because if no users are there, the right part of the ternary with the "fake" user object is used.

Reply:
Unfortunately in the beginning we have a situation when we don't have a user for a message, but we have nonempty users array. I tested it with two users.